.tool-versions

```diff
@@ -1,4 +1,4 @@
-nodejs 18.0.0
+nodejs 18.19.1
```

Comment:
18.18.0 is the minimum for eslint (not bumping the package cuz it will conflict with rn73 bump)